.github/workflows/publish-check.yml

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Based on Apache Arrow's java-nightly workflow
+# https://github.com/apache/arrow/blob/master/.github/workflows/java_nightly.yml
+name: Publish check
+
+on:
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  publish-check:
+    name: Publish check
+    runs-on: ubuntu-20.04
+    if: github.repository == 'apache/incubator-pekko'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: true
+
+      - name: Setup Java 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 11
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6
+
+      - name: Install Graphviz
+        run: |-
+          sudo apt-get install graphviz
+
+      - name: Publish to Apache Maven Local repo
+        run: sbt +publishM2
```

Comment:
So I had a look at this and you can replicate this by just using `sbt +compile:doc`, I confirmed locally on my laptop i.e.

```
[error]
[error]   last tree to typer: Ident(<argument>)
[error]        tree position: <unknown>
[error]             tree tpe: org.apache.pekko.cluster.sharding.typed.ShardingEnvelope[!?M]
[error]               symbol: <none>
[error]    symbol definition: <none> (a NoSymbol)
[error]       symbol package: <none>
[error]        symbol owners:
[error]            call site: package <root> in <none>
[error]
[error] == Source file context for tree position ==
[error]
[error]
[error] 	at scala.reflect.internal.SymbolTable.throwAssertionError(SymbolTable.scala:185)
[error] 	at scala.reflect.internal.SymbolTable.assert(SymbolTable.scala:162)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.findTemplateMaybe(ModelFactory.scala:825)
[error] 	at scala.tools.nsc.doc.model.MemberLookup.internalLink(MemberLookup.scala:27)
[error] 	at scala.tools.nsc.doc.model.MemberLookup.internalLink$(MemberLookup.scala:26)
[error] 	at scala.tools.nsc.doc.DocFactory$$anon$2.internalLink(DocFactory.scala:78)
[error] 	at scala.tools.nsc.doc.base.MemberLookupBase.$anonfun$memberLookup$6(MemberLookupBase.scala:72)
[error] 	at scala.collection.immutable.List.flatMap(List.scala:366)
[error] 	at scala.tools.nsc.doc.base.MemberLookupBase.memberLookup(MemberLookupBase.scala:72)
[error] 	at scala.tools.nsc.doc.base.MemberLookupBase.memberLookup$(MemberLookupBase.scala:61)
[error] 	at scala.tools.nsc.doc.DocFactory$$anon$2.memberLookup(DocFactory.scala:78)
[error] 	at scala.tools.nsc.doc.base.CommentFactoryBase.$anonfun$parseAtSymbol$16(CommentFactoryBase.scala:373)
[error] 	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:286)
[error] 	at scala.collection.immutable.Map$Map1.foreach(Map.scala:193)
[error] 	at scala.collection.TraversableLike.map(TraversableLike.scala:286)
[error] 	at scala.collection.TraversableLike.map$(TraversableLike.scala:279)
[error] 	at scala.collection.AbstractTraversable.map(Traversable.scala:108)
[error] 	at scala.tools.nsc.doc.base.CommentFactoryBase.linkedExceptions$1(CommentFactoryBase.scala:372)
[error] 	at scala.tools.nsc.doc.base.CommentFactoryBase.parse0$1(CommentFactoryBase.scala:390)
[error] 	at scala.tools.nsc.doc.base.CommentFactoryBase.parseAtSymbol(CommentFactoryBase.scala:417)
[error] 	at scala.tools.nsc.doc.base.CommentFactoryBase.parseAtSymbol$(CommentFactoryBase.scala:208)
[error] 	at scala.tools.nsc.doc.DocFactory$$anon$2.parseAtSymbol(DocFactory.scala:78)
[error] 	at scala.tools.nsc.doc.model.CommentFactory.parse(CommentFactory.scala:92)
[error] 	at scala.tools.nsc.doc.model.CommentFactory.parse$(CommentFactory.scala:90)
[error] 	at scala.tools.nsc.doc.DocFactory$$anon$2.parse(DocFactory.scala:78)
[error] 	at scala.tools.nsc.doc.model.CommentFactory.defineComment(CommentFactory.scala:82)
[error] 	at scala.tools.nsc.doc.model.CommentFactory.defineComment$(CommentFactory.scala:45)
[error] 	at scala.tools.nsc.doc.DocFactory$$anon$2.defineComment(DocFactory.scala:78)
[error] 	at scala.tools.nsc.doc.model.CommentFactory.$anonfun$comment$1(CommentFactory.scala:38)
[error] 	at scala.collection.mutable.HashMap.getOrElseUpdate(HashMap.scala:86)
[error] 	at scala.tools.nsc.doc.model.CommentFactory.comment(CommentFactory.scala:38)
[error] 	at scala.tools.nsc.doc.model.CommentFactory.comment$(CommentFactory.scala:36)
[error] 	at scala.tools.nsc.doc.DocFactory$$anon$2.comment(DocFactory.scala:78)
[error] 	at scala.tools.nsc.doc.model.ModelFactoryImplicitSupport.makeImplicitConversions(ModelFactoryImplicitSupport.scala:107)
[error] 	at scala.tools.nsc.doc.model.ModelFactoryImplicitSupport.makeImplicitConversions$(ModelFactoryImplicitSupport.scala:90)
[error] 	at scala.tools.nsc.doc.DocFactory$$anon$2.makeImplicitConversions(DocFactory.scala:78)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$DocTemplateImpl.<init>(ModelFactory.scala:360)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$$anon$6.<init>(ModelFactory.scala:656)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$.createDocTemplate$1(ModelFactory.scala:656)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$.createTemplate(ModelFactory.scala:703)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.makeMember0$1(ModelFactory.scala:797)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.makeMember(ModelFactory.scala:810)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$DocTemplateImpl.$anonfun$ownMembers$1(ModelFactory.scala:370)
[error] 	at scala.collection.immutable.List.flatMap(List.scala:366)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$DocTemplateImpl.<init>(ModelFactory.scala:370)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$PackageImpl.<init>(ModelFactory.scala:479)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$$anon$8.<init>(ModelFactory.scala:689)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$.createTemplate(ModelFactory.scala:689)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.makeMember0$1(ModelFactory.scala:797)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.makeMember(ModelFactory.scala:810)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$DocTemplateImpl.$anonfun$ownMembers$1(ModelFactory.scala:370)
[error] 	at scala.collection.immutable.List.flatMap(List.scala:366)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$DocTemplateImpl.<init>(ModelFactory.scala:370)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$PackageImpl.<init>(ModelFactory.scala:479)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$$anon$8.<init>(ModelFactory.scala:689)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$.createTemplate(ModelFactory.scala:689)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.makeMember0$1(ModelFactory.scala:797)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.makeMember(ModelFactory.scala:810)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$DocTemplateImpl.$anonfun$ownMembers$1(ModelFactory.scala:370)
[error] 	at scala.collection.immutable.List.flatMap(List.scala:366)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$DocTemplateImpl.<init>(ModelFactory.scala:370)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$PackageImpl.<init>(ModelFactory.scala:479)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$$anon$8.<init>(ModelFactory.scala:689)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$.createTemplate(ModelFactory.scala:689)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.makeMember0$1(ModelFactory.scala:797)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.makeMember(ModelFactory.scala:810)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$DocTemplateImpl.$anonfun$ownMembers$1(ModelFactory.scala:370)
[error] 	at scala.collection.immutable.List.flatMap(List.scala:366)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$DocTemplateImpl.<init>(ModelFactory.scala:370)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$PackageImpl.<init>(ModelFactory.scala:479)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$$anon$8.<init>(ModelFactory.scala:689)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$.createTemplate(ModelFactory.scala:689)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.makeMember0$1(ModelFactory.scala:797)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.makeMember(ModelFactory.scala:810)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$DocTemplateImpl.$anonfun$ownMembers$1(ModelFactory.scala:370)
[error] 	at scala.collection.immutable.List.flatMap(List.scala:366)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$DocTemplateImpl.<init>(ModelFactory.scala:370)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$PackageImpl.<init>(ModelFactory.scala:479)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$$anon$8.<init>(ModelFactory.scala:689)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$.createTemplate(ModelFactory.scala:689)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.makeMember0$1(ModelFactory.scala:797)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.makeMember(ModelFactory.scala:810)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$DocTemplateImpl.$anonfun$ownMembers$1(ModelFactory.scala:370)
[error] 	at scala.collection.immutable.List.flatMap(List.scala:366)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$DocTemplateImpl.<init>(ModelFactory.scala:370)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$PackageImpl.<init>(ModelFactory.scala:479)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$$anon$8.<init>(ModelFactory.scala:689)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$.createTemplate(ModelFactory.scala:689)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.makeMember0$1(ModelFactory.scala:797)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.makeMember(ModelFactory.scala:810)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$DocTemplateImpl.$anonfun$ownMembers$1(ModelFactory.scala:370)
[error] 	at scala.collection.immutable.List.flatMap(List.scala:366)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$DocTemplateImpl.<init>(ModelFactory.scala:370)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$PackageImpl.<init>(ModelFactory.scala:479)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$RootPackageImpl.<init>(ModelFactory.scala:487)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$$anon$7.<init>(ModelFactory.scala:671)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$.createTemplate(ModelFactory.scala:671)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$modelCreation$.createRootPackage(ModelFactory.scala:619)
[error] 	at scala.tools.nsc.doc.model.ModelFactory$$anon$1.<init>(ModelFactory.scala:55)
[error] 	at scala.tools.nsc.doc.model.ModelFactory.makeModel(ModelFactory.scala:52)
[error] 	at scala.tools.nsc.doc.DocFactory.makeUniverse(DocFactory.scala:91)
[error] 	at scala.tools.nsc.doc.DocFactory.generate$1(DocFactory.scala:135)
[error] 	at scala.tools.nsc.doc.DocFactory.document(DocFactory.scala:142)
[error] 	at xsbt.Runner.run(ScaladocBridge.scala:43)
[error] 	at xsbt.ScaladocBridge.run(ScaladocBridge.scala:21)
[error] 	at sbt.internal.inc.AnalyzingCompiler.doc(AnalyzingCompiler.scala:154)
[error] 	at sbt.internal.inc.AnalyzingCompiler.doc(AnalyzingCompiler.scala:133)
[error] 	at sbt.Doc$.$anonfun$scaladoc$1(Doc.scala:53)
[error] 	at sbt.Doc$.$anonfun$scaladoc$1$adapted(Doc.scala:41)
[error] 	at sbt.RawCompileLike$.$anonfun$prepare$1(RawCompileLike.scala:80)
[error] 	at sbt.RawCompileLike$.$anonfun$prepare$1$adapted(RawCompileLike.scala:73)
[error] 	at sbt.RawCompileLike$.$anonfun$cached$4(RawCompileLike.scala:64)
[error] 	at sbt.RawCompileLike$.$anonfun$cached$4$adapted(RawCompileLike.scala:62)
[error] 	at sbt.util.Tracked$.$anonfun$inputChangedW$1(Tracked.scala:220)
[error] 	at sbt.RawCompileLike$.$anonfun$cached$1(RawCompileLike.scala:69)
[error] 	at sbt.RawCompileLike$.$anonfun$cached$1$adapted(RawCompileLike.scala:53)
[error] 	at sbt.Defaults$.$anonfun$docTaskSettings$4(Defaults.scala:2164)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:63)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:69)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:283)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:24)
[error] 	at sbt.Execute.work(Execute.scala:292)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:283)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:65)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[error] 	at java.base/java.lang.Thread.run(Thread.java:829)
[error] (cluster-sharding-typed / Compile / doc) java.lang.AssertionError: assertion failed:
[error]
[error]      while compiling: /Users/mdedetrich/github/incubator-pekko/cluster-sharding-typed/src/main/scala/org/apache/pekko/cluster/sharding/typed/delivery/internal/ShardingProducerControllerImpl.scala
[error]         during phase: globalPhase=terminal, enteringPhase=typer
[error]      library version: version 2.12.18
[error]     compiler version: version 2.12.18
[error]   reconstructed args: -bootclasspath /Users/mdedetrich/.sbt/boot/scala-2.12.18/lib/scala-library.jar -sourcepath /Users/mdedetrich/github/incubator-pekko -classpath /Users/mdedetrich/github/incubator-pekko/actor-typed/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/actor-typed/target/scala-2.12/CompileJdk9-classes:/Users/mdedetrich/github/incubator-pekko/actor/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/slf4j/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/cluster-typed/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/cluster/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/remote/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/stream/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/protobuf-v3/target/scala-2.12/stripped/stripped/pekko-protobuf-v3-assembly-1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT.jar:/Users/mdedetrich/github/incubator-pekko/pki/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/coordination/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/cluster-tools/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/distributed-data/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/cluster-sharding/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/persistence/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/cluster-sharding/target/scala-2.12/CompileJdk9-classes:/Users/mdedetrich/github/incubator-pekko/persistence-typed/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/stream-typed/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/persistence-query/target/scala-2.12/classes:/Users/mdedetrich/github/incubator-pekko/remote/target/scala-2.12/CompileJdk9-classes:/Users/mdedetrich/github/incubator-pekko/protobuf-v3/target/scala-2.12/stripped/pekko-protobuf-v3-assembly-1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/slf4j/slf4j-api/1.7.36/slf4j-api-1.7.36.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/agrona/agrona/1.19.2/agrona-1.19.2.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/modules/scala-java8-compat_2.12/1.0.2/scala-java8-compat_2.12-1.0.2.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/typesafe/config/1.4.2/config-1.4.2.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/lmdbjava/lmdbjava/0.7.0/lmdbjava-0.7.0.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/reactivestreams/reactive-streams/1.0.4/reactive-streams-1.0.4.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/typesafe/ssl-config-core_2.12/0.6.1/ssl-config-core_2.12-0.6.1.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/hierynomus/asn-one/0.5.0/asn-one-0.5.0.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/github/jnr/jnr-ffi/2.1.9/jnr-ffi-2.1.9.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/github/jnr/jffi/1.2.18/jffi-1.2.18.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/github/jnr/jffi/1.2.18/jffi-1.2.18-native.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/github/jnr/jnr-constants/0.9.12/jnr-constants-0.9.12.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/ow2/asm/asm/5.0.3/asm-5.0.3.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/ow2/asm/asm-commons/5.0.3/asm-commons-5.0.3.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/ow2/asm/asm-analysis/5.0.3/asm-analysis-5.0.3.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/ow2/asm/asm-tree/5.0.3/asm-tree-5.0.3.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/ow2/asm/asm-util/5.0.3/asm-util-5.0.3.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/github/jnr/jnr-a64asm/1.0.0/jnr-a64asm-1.0.0.jar:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/github/jnr/jnr-x86asm/1.0.2/jnr-x86asm-1.0.2.jar -d /Users/mdedetrich/github/incubator-pekko/cluster-sharding-typed/target/scala-2.12/api -doc-title Apache Pekko -doc-version 1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-source-url https://github.com/apache/incubator-pekko/tree/main/€{FILE_PATH_EXT}#L€{FILE_LINE} -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/modules/scala-java8-compat_2.12/1.0.2/scala-java8-compat_2.12-1.0.2.jar#https://javadoc.io/page/org.scala-lang.modules/scala-java8-compat_2.12/1.0.2/ -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/cluster-tools/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/modules/java.base#https://docs.oracle.com/en/java/javase/11/docs/api/java.base/ -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/github/jnr/jffi/1.2.18/jffi-1.2.18-native.jar#https://javadoc.io/page/com.github.jnr/jffi/1.2.18/ -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/slf4j/slf4j-api/1.7.36/slf4j-api-1.7.36.jar#https://javadoc.io/page/org.slf4j/slf4j-api/1.7.36/ -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/actor-typed/target/scala-2.12/CompileJdk9-classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/github/jnr/jnr-ffi/2.1.9/jnr-ffi-2.1.9.jar#https://javadoc.io/page/com.github.jnr/jnr-ffi/2.1.9/ -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/github/jnr/jnr-a64asm/1.0.0/jnr-a64asm-1.0.0.jar#https://javadoc.io/page/com.github.jnr/jnr-a64asm/1.0.0/ -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/coordination/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/reactivestreams/reactive-streams/1.0.4/reactive-streams-1.0.4.jar#https://javadoc.io/page/org.reactivestreams/reactive-streams/1.0.4/ -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/ow2/asm/asm-analysis/5.0.3/asm-analysis-5.0.3.jar#https://javadoc.io/page/org.ow2.asm/asm-analysis/5.0.3/ -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/protobuf-v3/target/scala-2.12/stripped/stripped/pekko-protobuf-v3-assembly-1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT.jar#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/typesafe/ssl-config-core_2.12/0.6.1/ssl-config-core_2.12-0.6.1.jar#https://javadoc.io/page/com.typesafe/ssl-config-core_2.12/0.6.1/ -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/ow2/asm/asm-util/5.0.3/asm-util-5.0.3.jar#https://javadoc.io/page/org.ow2.asm/asm-util/5.0.3/ -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/pki/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/persistence-typed/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/stream-typed/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/stream/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/ow2/asm/asm-commons/5.0.3/asm-commons-5.0.3.jar#https://javadoc.io/page/org.ow2.asm/asm-commons/5.0.3/ -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/actor/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/slf4j/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/cluster/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/agrona/agrona/1.19.2/agrona-1.19.2.jar#https://javadoc.io/page/org.agrona/agrona/1.19.2/ -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/remote/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/ow2/asm/asm-tree/5.0.3/asm-tree-5.0.3.jar#https://javadoc.io/page/org.ow2.asm/asm-tree/5.0.3/ -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/github/jnr/jnr-x86asm/1.0.2/jnr-x86asm-1.0.2.jar#https://javadoc.io/page/com.github.jnr/jnr-x86asm/1.0.2/ -doc-external-doc:/Users/mdedetrich/.sbt/boot/scala-2.12.18/lib/scala-library.jar#https://www.scala-lang.org/api/2.12.18/ -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/cluster-typed/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/cluster-sharding/target/scala-2.12/CompileJdk9-classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/github/jnr/jffi/1.2.18/jffi-1.2.18.jar#https://javadoc.io/page/com.github.jnr/jffi/1.2.18/ -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/persistence/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/actor-typed/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/remote/target/scala-2.12/CompileJdk9-classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/github/jnr/jnr-constants/0.9.12/jnr-constants-0.9.12.jar#https://javadoc.io/page/com.github.jnr/jnr-constants/0.9.12/ -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/cluster-sharding/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/typesafe/config/1.4.2/config-1.4.2.jar#https://javadoc.io/page/com.typesafe/config/1.4.2/ -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/ow2/asm/asm/5.0.3/asm-5.0.3.jar#https://javadoc.io/page/org.ow2.asm/asm/5.0.3/ -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/persistence-query/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/hierynomus/asn-one/0.5.0/asn-one-0.5.0.jar#https://javadoc.io/page/com.hierynomus/asn-one/0.5.0/ -doc-external-doc:/Users/mdedetrich/github/incubator-pekko/distributed-data/target/scala-2.12/classes#https://pekko.apache.org/api/pekko/1.1.0-M0+107-ac1ee69f+20231013-1722-SNAPSHOT -doc-external-doc:/Users/mdedetrich/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/lmdbjava/lmdbjava/0.7.0/lmdbjava-0.7.0.jar#https://javadoc.io/page/org.lmdbjava/lmdbjava/0.7.0/ -implicits -diagrams -groups -doc-canonical-base-url https://pekko.apache.org/api/pekko/current/
[error]
[error]   last tree to typer: Ident(<argument>)
[error]        tree position: <unknown>
[error]             tree tpe: org.apache.pekko.cluster.sharding.typed.ShardingEnvelope[!?M]
[error]               symbol: <none>
[error]    symbol definition: <none> (a NoSymbol)
[error]       symbol package: <none>
[error]        symbol owners:
[error]            call site: package <root> in <none>
[error]
[error] == Source file context for tree position ==
[error]
[error] Total time: 196 s (03:16), completed 13 Oct 2023, 17:25:51
```

Note the

```
[error] (cluster-sharding-typed / Compile / doc) java.lang.AssertionError: assertion failed:
```

Which tells you which command fails.